scripts/pre_commit_linter.py

```diff
@@ -57,6 +58,10 @@
     '--path',
     help='path to the directory with files to be linted',
     action='store')
+_PARSER.add_argument(
+    '--silent',
```

Comment:
I'm not convinced you need to do this silencing. Without it, how many lines of output are there?